src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/utilsAktivitet.ts

```diff
@@ -0,0 +1,38 @@
+import { SelectOption } from '../../../../komponenter/Skjema/SelectMedOptions';
+import { Stønadstype } from '../../../../typer/behandling/behandlingTema';
+import { AktivitetType, AktivitetTypeTilTekst } from '../typer/vilkårperiode/aktivitet';
+
+export const aktivitetTypeTilTekst = (type: AktivitetType | '') => {
+    if (type === '') return type;
+
+    return AktivitetTypeTilTekst[type];
+};
+
+export const lagAktivitetTypeOptions = (stønadstype: Stønadstype): SelectOption[] => {
```

Comment:
Ikke veeeldig glad i `TypeOptions` som navn kjenner jeg. Kunne vi kanskje rebrandet denne som 
```suggestion
export const valgbareAktivitetTyper = (stønadstype: Stønadstype): SelectOption[] => {
```
eller noe sånt?